dashboard/app/config.go

```diff
@@ -475,6 +496,20 @@ func checkKernelRepos(ns string, repos []KernelRepo) {
 			panic(fmt.Sprintf("%v: bad kernel repo reporting priority %v for %q", ns, prio, repo.Alias))
 		}
 		checkCC(&repo.CC)
+		for _, label := range []string{repo.LabelIntroduced, repo.LabelReached} {
+			if label == "" {
```

Comment:
Does it make sense to check that there is at least 1 label/tag if TestOtherTrees is set? Or any other sanity checks? It looks like it's a bit non-trivial to configure and easy to misuse (if done by anybody other than you :)).